Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -8,7 +8,6 @@
 
 import UIKit
 import SafariServices
-import Crashlytics
 import ViewAnimator
 
 class NewsTableViewController: UITableViewController,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/e548a8ccaa17e8374ed5774dd45483a7224d0088/Headlines/src/Controllers/NewsTableViewController.swift#L13"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 275 lines</td>
    </tr>
  </tbody>
</table>